build.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# parameters $1 is build number
+
+export WORKSPACE=`pwd`
+export RPM_VERSION="0.8"
+export BUILD_NUMBER=$1
+
+error() {
+  local parent_lineno="$1"
+  local message="$2"
+  local code="${3:-1}"
+  if [[ -n "$message" ]] ; then
+    echo "Error on or near line ${parent_lineno}: ${message}; exiting with status ${code}"
+  else
+    echo "Error on or near line ${parent_lineno}; exiting with status ${code}"
+  fi
+  exit "${code}"
+}
+
+#build and run unit tests
+trap 'error ${LINENO}' ERR
+virtualenv venv
+source venv/bin/activate
```

Comment:
Yes it will fail with python 2.7. On the CentOS box, the virtualenv command is symlinked to a python3.3 version of vitrualenv, so it works there only. This is just a first pass at getting something in place for the build server to use.  If we want it to be more general purpose for other operating systems, etc., then I can make it more robust.